romi-rover/control_panel/test/mock/mock_timer.h

```diff
@@ -0,0 +1,14 @@
+#include "gmock/gmock.h"
+#include <IEventTimer.h>
+
+#ifndef __EVENT_BUTTON_H
```

Comment:
Is __EVENT_BUTTON_H correct? Just to make sure it's not a copy / paste that can come back and define out a real header!